cookbooks/bcpc/recipes/haproxy.rb

```diff
@@ -67,7 +67,8 @@
 template "/etc/haproxy/haproxy.cfg" do
   source "haproxy.cfg.erb"
   mode 00644
-  variables(:mysql_servers => get_nodes_for("mysql","bcpc"))
+  variables(:mysql_servers => get_nodes_for("mysql","bcpc"),
+    :oozie_servers => get_nodes_for("oozie", "bcpc-hadoop"))
```

Comment:
Just a question, but I wonder if there's any way we can break the `haproxy.cfg` up into pieces so that we can deploy solely HAProxy then deploy services incrementally? (E.g. an `/etc/haproxy/haproxy.d/<service>.cfg` using include semantics?) This would aide in testing components and HAProxy but not having to stand up a whole cluster.